ee/clickhouse/sql/person.py

```diff
@@ -201,6 +251,27 @@
 # Other queries
 #
 
+GET_TEAM_PERSON_DISTINCT_IDS = """
+SELECT distinct_id, argMax(person_id, _timestamp) as person_id
+FROM (
+    SELECT distinct_id, person_id, max(_timestamp) as _timestamp
+    FROM person_distinct_id
+    WHERE team_id = %(team_id)s
+    GROUP BY person_id, distinct_id, team_id
+    HAVING max(is_deleted) = 0
+)
+GROUP BY distinct_id
+"""
+
+# Query to query distinct ids using the new table. To be improved
```

Comment:
are we getting this in?

Reply:
Separate PR: https://github.com/PostHog/posthog/pull/7577

Caused a merge conflict for myself but /shrug